### PR TITLE
LPS-111667 Pass the number of responses through jsp

### PR DESCRIPTION
**Pull request by carolmariaabb**

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:forms

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 1]
```diff
@@ -21,6 +21,8 @@ String redirect = ParamUtil.getString(request, "redirect");
 
 DDMFormInstance formInstance = ddmFormAdminDisplayContext.getDDMFormInstance();
 
+int entriesCount = 0;
```
Comment (brunofarache) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:24]: rename to InstanceRecordsCount?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 2]
```diff
@@ -53,7 +61,7 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
 		navigationItems="<%= ddmFormAdminDisplayContext.getFormBuilderNavigationItems() %>"
 	/>
 
-	<nav class="management-bar management-bar-light navbar navbar-expand-md toolbar-group-field">
+	<nav class="management-bar management-bar-light navbar navbar-expand-md toolbar-group-field" id="ddmFormInstanceManagementToolbar">
```
Comment (brunofarache) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:64]: missing <portlet:namespace /> on id.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 3]
```diff
@@ -141,6 +149,10 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
 
 <liferay-portlet:resourceURL copyCurrentRenderParameters="<%= false %>" id="saveFormInstance" var="autoSaveFormInstanceURL" />
 
+<div class="hide" id="viewFormInstanceRecords">
```
Comment (brunofarache) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:152]: missing <portlet:namespace /> on id.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 4]
```diff
@@ -141,6 +149,10 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
...
+	<liferay-util:include page="/admin/list_form_instance_records.jsp" servletContext="<%= application %>" />
```
Comment (brunofarache) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:153]: never seen `servletContext` I need to understand what this is for.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js [hunk 5]
```diff
@@ -333,6 +333,12 @@ class Form extends Component {
 		return ruleBuilderVisible && this.isFormBuilderView();
 	}
 
+	isShowEntries() {
+		const {entriesVisible} = this.state;
```
Comment (brunofarache) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js:337]: rename to showEntries

Comment (brunofarache) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js:337]: it's probably better to create a feature flag to show/hide the entries tab, we will hide it until Reports is completely implemented.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js [hunk 6]
```diff
@@ -555,13 +565,16 @@ class Form extends Component {
 		submitForm(document.querySelector(`#${namespace}editForm`));
 	}
 
-	syncRuleBuilderVisible(visible) {
+	syncTabsVisible(entriesVisible, ruleBuilderVisible) {
```
Comment (brunofarache) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js:568]: syncVisibleTab

**Review by brunofarache — CHANGES_REQUESTED** [review]

**Comment by brunofarache** [comment]
Just started reviewing :)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp (hunk 4 above)
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:153]: "The servletContext refers to the request context that the included JSP should use. Passing <%= application %> to this attribute lets the included JSP use the same request object as other objects that might be set in the prior JSP. "

https://help.liferay.com/hc/pt/articles/360023185391-Using-Liferay-Util-Include-

Example of usage [here](https://github.com/liferay/liferay-portal/blob/43f034138129c4601c1dd0cdf076d8cfc4e99c01/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view.jsp#L29).

**Comment by brunofarache** [comment]
Pull request submitted to https://github.com/brunofarache/liferay-portal/pull/163. [See changes here.](https://github.com/brunofarache/liferay-portal/compare/98ae70fbd4a7~5...98ae70fbd4a7)

**Comment by brunofarache** [comment]
Pull request submitted to https://github.com/brunofarache/liferay-portal/pull/165. [See changes here.](https://github.com/brunofarache/liferay-portal/compare/1ad3338c5be7~6...1ad3338c5be7)

**Pull request closed without merging**